db/version_edit.cc

```diff
@@ -186,6 +186,13 @@ bool VersionEdit::EncodeTo(std::string* dst) const {
     PutVarint32(dst, NewFileCustomTag::kFileChecksumFuncName);
     PutLengthPrefixedSlice(dst, Slice(f.file_checksum_func_name));
 
+    if (f.max_timestamp != kDisableUserTimestamp) {
+      assert(f.min_timestamp != kDisableUserTimestamp);
```

Comment:
Can we return `false` if assertion fails?

Reply:
sure, fixed